src/chi/components/alert/_bubble.scss

```diff
@@ -3,64 +3,43 @@
 .chi {
   .m-alert {
     &.-bubble {
+      align-items: center;
+      justify-content: start;
 
-      & > .-close {
-        position: absolute;
-        right: 1rem;
-        top: 50%;
-        transform: translateY(-50%);
+      &.-dismiss {
+        justify-content: space-between;
       }
 
-      &.-small {
-        &::before {
-          display: none;
-        }
-
-        & > .-close {
-          top: 0.3rem;
-          transform: initial;
+      & > .-close {
+        &.-small {
+          line-height: 0.75rem;
+          padding: 0.25rem;
         }
       }
 
       &.-large {
         padding-bottom: 1.1875rem;
         padding-top: 0.75rem;
 
-        & > .-close {
-          top: 0.9375rem;
-          transform: initial;
-        }
+        & .m-alert__content {
+          & .a-icon {
+            align-self: start;
+            top: 0.1875rem;
+          }
 
-        &::before {
-          font-size: $text-large;
-          margin-bottom: 0.4375rem;
-          padding: 0.25rem 0;
+          & > div:not(p) {
+            text-transform: capitalize;
+          }
         }
-      }
-
-      @each $state in map-keys($states) {
-        &.-#{$state} {
-          &.-hasIcon {
-            &.-small {
-              padding-left: 2.4375rem;
 
-              &::after {
-                height: 1rem;
-                left: 0.9375rem;
-                width: 1rem;
-              }
-            }
+        &.-dismiss {
```

Comment:
@davidojedacl this class can be deleted as it is inheriting the same flex position properties that .chi .m-alert.-bubble.-dismiss